src/kl_exchange.jl

```diff
@@ -16,6 +16,78 @@ function d_criterion(model_matrix;
         (1 / size(model_matrix, 2))
 end
 
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of A-optimality.
+
+"""
+function a_criterion(model_matrix;
+                     tolerance = 1e-9)
+    M = model_matrix' * model_matrix + (I * tolerance)
+    C = cholesky(M, check = false)
+    if !issuccess(C) return 0 end
+    M⁻¹ = inv(C)
+    size(model_matrix, 2)/tr(M⁻¹)/size(model_matrix, 1)
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of T-optimality. Experimental.
+
+"""
+function t_criterion(model_matrix;
+                     tolerance = 0)
+    M = model_matrix' * model_matrix + (I * tolerance)
+    tr(M)/size(model_matrix, 1)/size(model_matrix, 2)
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+Criterion of G-optimality. Experimental.
```

Comment:
Is this an approximation of the G-criterion or a closed formula?

Reply:
This criterion minimize the maximum value of main diagonal of the hat matrix  `X(X'X)⁻¹}X'`.  I have no better ref than: https://www.itl.nist.gov/div898/handbook/pri/section5/pri52.htm